src/Counter.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export class Counter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      count: 1,
+    };
+  }
+
+  increment = () => {
+    this.setState(prevState => (
+      { count: prevState.count + 1 }
+    ));
```

Comment:
так это не переносится
```suggestion
    this.setState(prevState => ({
      count: prevState.count + 1
    }));
    
```